airbyte-workers/src/test/java/io/airbyte/workers/process/DockerProcessFactoryTest.java

```diff
@@ -75,7 +75,7 @@ public void testImageExists() throws IOException, WorkerException {
     Path workspaceRoot = Files.createTempDirectory(Files.createDirectories(TEST_ROOT), "process_factory");
 
     final DockerProcessFactory processFactory = new DockerProcessFactory(workspaceRoot, "", "", "");
-    assertTrue(processFactory.checkImageExists("airbyte/scheduler:dev"));
+    assertTrue(processFactory.checkImageExists("busybox"));
```

Comment:
we are getting rid of the dev images since no one really uses there, and it's confusing to have two potential source of truths for the dev images.